metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -5,13 +5,13 @@
    [clojure.java.jdbc :as j]
    [clojure.set :as set]
    [clojure.string :as str]
-   [cmr.common.concepts :as cc]
+   [cmr.common.concepts :as common-concepts]
```

Comment:
@oubiwann That's actually a standard now, unless the namespace alias is very widely used across the code.